File: compare/loop.php
<?php

function test($x)
{
    $t1 = microtime(true);
    $a = 0;
    for($i = 0; $i < $x; $i++)
    {
        $a++;
    }
    $t2 = microtime(true);

    echo "Time for $x was " . ($t2 - $t1) . "\n";

    return $a;
}


echo test(10000000000);
?>

File: compare/loop.py
import time

def test(x):
    a = 0

    #now you count only the loop time
    t1 = time.time()
    while(a < x):
        a += 1

    t2 = time.time()

    print("Time for {} was {}".format(x, t2 - t1))
    return a

print(test(100000000))
